src/app/clusters/time-synchronization-server/time-synchronization-server.cpp

```diff
@@ -663,6 +670,17 @@ void TimeSynchronizationServer::ClearEventFlag(TimeSyncEventFlag flag)
     mEventFlag        = static_cast<TimeSyncEventFlag>(eventFlag);
 }
 
+void TimeSynchronizationServer::OnFabricRemoved(const FabricTable & fabricTable, FabricIndex fabricIndex)
+{
+    if (!mTrustedTimeSource.IsNull() && mTrustedTimeSource.Value().fabricIndex == fabricIndex)
+    {
+        DataModel::Nullable<Structs::TrustedTimeSourceStruct::Type> tts;
+        tts.SetNull();
```

Comment:
The `Nullable` datatype is set to null by the constructor, so I think that explicit `SetNull()` this is not required here.